ext/opentelemetry-ext-aiopg/src/opentelemetry/ext/aiopg/wrappers.py

```diff
@@ -0,0 +1,231 @@
+# Copyright The OpenTelemetry Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""
+The trace integration with aiopg based on dbapi integration,
+where replaced sync wrap methods to async
+
+Usage
+-----
+
+.. code-block:: python
+
+    from opentelemetry import trace
+    from opentelemetry.ext.aiopg import trace_integration
+    from opentelemetry.trace import TracerProvider
+
+    trace.set_tracer_provider(TracerProvider())
+
+    trace_integration(aiopg.connection, "_connect", "postgresql", "sql")
+
+API
+---
+"""
+import logging
+import typing
+
+import wrapt
+
+from opentelemetry.ext.aiopg.aiopg_integration import (
+    AiopgIntegration,
+    get_traced_connection_proxy,
+)
+from opentelemetry.ext.aiopg.version import __version__
+from opentelemetry.instrumentation.utils import unwrap
+from opentelemetry.trace import Tracer, TracerProvider, get_tracer
+
+logger = logging.getLogger(__name__)
+
+
+def trace_integration(
+    connect_module: typing.Callable[..., typing.Any],
+    connect_method_name: str,
+    database_component: str,
+    database_type: str = "",
+    connection_attributes: typing.Dict = None,
+    tracer_provider: typing.Optional[TracerProvider] = None,
+):
+    """Integrate with aiopg library.
+        based on dbapi integration, where replaced sync wrap methods to async
+
+        Args:
+            connect_module: Module name where connect method is available.
+            connect_method_name: The connect method name.
+            database_component: Database driver name or
+                database name "postgreSQL".
+            database_type: The Database type. For any SQL database, "sql".
+            connection_attributes: Attribute names for database, port, host and
+                user in Connection object.
+            tracer_provider: The :class:`opentelemetry.trace.TracerProvider` to
+                use. If ommited the current configured one is used.
+    """
+    tracer = get_tracer(__name__, __version__, tracer_provider)
+    wrap_connect(
+        tracer,
+        connect_module,
+        connect_method_name,
+        database_component,
+        database_type,
+        connection_attributes,
+    )
+
+
+def wrap_connect(
+    tracer: Tracer,
+    connect_module: typing.Callable[..., typing.Any],
+    connect_method_name: str,
+    database_component: str,
+    database_type: str = "",
+    connection_attributes: typing.Dict = None,
+):
```

Comment:
If `wrap_connect` is for aiopg only, why do we need to pass `connect_module` and `connect_method_name` as parameters? same for wrap_pool

Reply:
done